app/helpers/validation_helper.rb

```diff
@@ -44,7 +44,7 @@ def error_messages_for(object)
   def error_messages_formatted(errors, intro = "")
     return unless errors.present?
 
-    error_messages = errors.map { |msg| content_tag(:li, msg.gsub(/^(.*?)\^/, "").html_safe) }
+    error_messages = errors.map { |msg| content_tag(:li, msg.sub(/^[\w\s]+\^/, "").html_safe) }
```

Comment:
```suggestion
    error_messages = errors.map { |msg| content_tag(:li, msg.sub(/^[\w\s]*\^/, "").html_safe) }
```
We can also get rid of the attribute name at the start of the message by setting the message format to `"%{message}"`, so we may accidentally end up with a message that uses a `^` to suppress the attribute, but also has message format on the model or attribute that does the same thing so the message we're getting here would just start with a `^`. That duplicate formatting would be coding mistake, but I'd prefer if in that case we still don't end up with the `^` in the output

Reply:
That makes sense, thank you!